### ISSUE TITLE
Feature: "request desktop version"

### ISSUE THREAD

**Issue by doniks** [ubports/morph-browser#324]
Firefox on Android has a very helpful menu: "Request desktop site"

You can tick it in a tab and then the website switches from mobile view to desktop view. For example on github. I find it to be a very helpful for misbehaving websites. E.g., things that are not (easily) possible on a mobile site. 

NB: Yes, some websites have their own toggles to switch to desktop mode, but then I sometimes had problems finding a way to switch back.

**Comment by UniversalSuperBox** [comment]
Hi @doniks,

It is possible to set desktop mode browser-wide by checking "Set Desktop Mode" in Morph's Settings. Is this what you are looking for?

**Comment by doniks** [comment]
I see. Well, I guess similar to this, but a)easier to access, and b) not global, but local to one tab.

My use case goes like this: 
* in general I'm happy with mobile view
* approx once a month, I'm stuck on some website, where I cant do one particular action
* I request desktop mode
* perform this one action
* switch back to mobile

ps: just tried the setting on git hub, it didn't give me the desktop view, even after reload. but I guess that is a separate issue

**Comment by mateosalta** [comment]
Yes, I too would like a one tab only quick toggle for desktop mode,  going into settings to turn on off desktop mode for one page is too much I think.

**Comment by mateosalta** [comment]
This is probably related: https://github.com/ubports/morph-browser/issues/157

**Comment by niutech** [comment]
The option to override user agent string per domain was already merged in #203, but it was disabled by the commit [2cd10b4](https://github.com/ubports/morph-browser/commit/2cd10b497c704dc60ea8e6a7407e832e2a1dc28f) by @balcy. Why was it so and could it be enabled now?

**Comment by balcy** [comment]
it has not been merged, but removed before the merge because the user agent could not be changed reliably for QtWebEngine 5.11 (see https://github.com/ubports/morph-browser/pull/203#issuecomment-506968966) 

With https://github.com/ubports/morph-browser/pull/368 they are tried to be reactivated (the new QtWebEngine should handle them correctly)

**Comment by niutech** [comment]
@balcy thank you for your fix, hopefully it will be merged soon!
